docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -321,7 +321,7 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
 |`table`|The table which contains the key value pairs|Yes||
 |`keyColumn`|The column in `table` which contains the keys|Yes||
 |`valueColumn`|The column in `table` which contains the values|Yes||
-|`filter`|The filter to use when selecting lookups, this is is used to create a where clause on lookup population|No| No Used|
+|`filter`|The filter to use when selecting lookups, this is is used to create a where clause on lookup population|No| Not Used|
 |`tsColumn`| The column in `table` which contains when the key was updated|No|Not used|
```

Comment:
Hey Charles, after giving it a couple of minutes of thought (probs should have done this before commenting) I understand why it's doing this as it needs to be able to handle deleted entries etc which would not happen if it just pulled the updated entries based on timestamp as per the documentation. Would end up having to do a big in query with the set it already had to work out the deleted ones etc, so not sure it's worth it. Still want me to open an issue or just leave it?